_docs/backenders/strategy-2021/roles/quality.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: Quality
+parent: Roles
+grand_parent: Strategy 2021
+---
+
+# Quality role definition
+
+Delivering high-quality work is a trademark of Vizzuality. Ensuring clients and
+users are happy using the products we build is something we need to pay 
+attention to in everything we do daily. By delivering high-quality work, on time 
+and efficiently, we keep clients happy, and we can be proud of the work we do.
+
+This role is the point of contact between the Backend and ongoing Quality 
+initiatives at Vizzuality.
+
+**Current responsible**: Henrique Pacheco and Alex Larrañaga
+
+## Responsibilities
+
+- External point of contact and team representative for ongoing Quality 
+  initiatives at Vizzuality:
+    - Attend the monthly Quality initiative meetings.
+    - Share a summary and relevant details of these meetings with the BE FA.
+- Prioritize actions to be taken in terms of Quality, considering their cost 
+  and level of effort.
```

Comment:
(not a blocker for approval, can be dealt with later) Should we clarify this a bit? I think this could mean something like we did earlier on this year when we agreed on the key aspects of quality-oriented work the BE group wanted to focus on, but (also linked to the last item in the list about accomplishments) should also include coordinating the choice of OKRs (again, something we did already).

The way we've written this may sound like we expect the role holder to do the prioritization, though I imagine we want more of a facilitation/moving-things-forward responsibility.

```suggestion
- Ensure the team has a clear focus on quality:
  - Support the FA engineers through the choice and periodic revision of actions
    to be taken in terms of Quality, considering their cost and level of effort.
  - Help set metrics for tracking the impact of quality-focused interventions on
    the quality of our work.
```

Reply:
yes, 100% agreed, thanks for the suggestion @hotzevzl :bow: